src/models/ContractEntity.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+/* eslint-disable class-methods-use-this */
+
+import BigNumber from 'bignumber.js';
+import Comparable from "../observer/Comparable";
+
+/**
+ * Represents model object.
+ */
+export default class ContractEntity extends Comparable<ContractEntity> {
+  public contractAddress: string;
+
+  public entityType?: string;
```

Comment:
entityType is a mandatory field here. we don't need to add "?" as optional field?
Same for timestamp.
created_at, updated_at are optional fields during object creating.